presto-hive/src/main/java/com/facebook/presto/hive/OrcFileWriterConfig.java

```diff
@@ -54,6 +55,7 @@
     private boolean isStringDictionaryEncodingEnabled = OrcWriterOptions.DEFAULT_STRING_DICTIONARY_ENCODING_ENABLED;
     private boolean isStringDictionarySortingEnabled = OrcWriterOptions.DEFAULT_STRING_DICTIONARY_SORTING_ENABLED;
     private boolean isFlatMapWriterEnabled = DEFAULT_FLAT_MAP_WRITER_ENABLED;
+    private boolean addHostnameToFileMetadata;
```

Comment:
nit : rename the var as addHostnameToFileMetadataEnabled so the getter reads a bit nicer.
`isAddHostnameToFileMetadataEnabled` reads better than `isAddHostnameToFileMetadata`

Reply:
Done. Renamed both the field and the config key name.